.github/workflows/log-coderabbit-review.yml

```
@@ -0,0 +1,47 @@
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.pull_request.head.ref }}
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Append CodeRabbit review to .claude/S&P.md
+        env:
+          REVIEW_BODY: ${{ github.event.review.body }}
+          PR_NUMBER: ${{ github.event.pull_request.number }}
+          PR_TITLE: ${{ github.event.pull_request.title }}
+          REVIEW_DATE: ${{ github.event.review.submitted_at }}
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC1132:warning:11:15: This & terminates the command. Escape it or add space after & to silence

(shellcheck)